CONTRIBUTING.md

```diff
@@ -63,12 +128,98 @@ $ npm test -- --config jest.config.json
 # etc
 ```
 
-### Run coverage
+#### Run coverage
+
+As the Boy Scout Rule states, you are expected to leave the code better than you found it. To do so, please check that your code is properly covered by meaningful tests.
 
 ```sh
 npm test -- --coverage
 ```
 
+#### Browser testing
+
+You can find the list of supported browser in the [List of supported browsers](./README.md#list-of-supported-browsers) section of the README.
+
+If you are developping on a Mac, you will find some Virtual Machines to help you test, on [Microsoft Edge Tools](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) webiste.
+
+### Documentation
+
+After working on a component, you are expected to update the documentation.
+
+Documentation can be found in the [Design System](https://github.com/AxaGuilDEv/design-system) repository.
+
+Do not forget to update the component status page, located in this repository at:
+
+```text
+└─ src/
+    └─ pages/
+      └─ get-started/
+          └─ component-status/
+            └─ markdown/
+                └─ content.md
+```
+
+Direct link (on master branch): [content.md](https://github.com/AxaGuilDEv/design-system/blob/master/src/pages/get-started/component-status/markdown/content.md)
+
+### Commit messages
+
+We are using the [Angular commit message format](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format), you are expected to follow it. This is very important because our changlog is generated using commit messages.
+
+Keep in mind, though, that your pull request will be squashed into master, so repository mainteners may use your commit message but are not entitled to use them as is.
+
+## Scripts
```

Comment:
Pour moi c'est de trop car les scripts évoluent et si le dev veut les voir il suffit d'aller sur le package.json

Reply:
Not everybody is a jedi master. Think of this as a way to help newcomers.

Reply:
I agree with Gilles, we never have too much documentation